DisplayOptionsPanel.c

```diff
@@ -116,8 +116,14 @@ DisplayOptionsPanel* DisplayOptionsPanel_new(Settings* settings, ScreenManager*
    Panel_add(super, (Object*) CheckItem_newByRef("Add guest time in CPU meter percentage", &(settings->accountGuestInCPUMeter)));
    Panel_add(super, (Object*) CheckItem_newByRef("Also show CPU percentage numerically", &(settings->showCPUUsage)));
    Panel_add(super, (Object*) CheckItem_newByRef("Also show CPU frequency", &(settings->showCPUFrequency)));
-   #ifdef HAVE_SENSORS_SENSORS_H
-   Panel_add(super, (Object*) CheckItem_newByRef("Also show CPU temperature (requires libsensors)", &(settings->showCPUTemperature)));
+   #ifdef BUILD_WITH_CPU_TEMP
+   Panel_add(super, (Object*) CheckItem_newByRef(
+   #ifdef HTOP_LINUX
```

Comment:
This one (`#ifdef`) is inconsistent with the next case (`#if defined`), it'd be easier to read if both used the same preprocessor directive.

Reply:
You mean:
```suggestion
   #if defined(HTOP_LINUX)
```